CONTRIBUTING.md

```diff
@@ -1,44 +1,55 @@
-# Contributing to _xikolo-ios_
+# How to Contribute
 
 ### I want to report a problem or ask a question
 Before submitting a new GitHub issue, please make sure to check the [existing GitHub issues](https://github.com/openHPI/xikolo-ios/issues). If this doesn't help, please [submit an issue](https://github.com/openHPI/xikolo-ios/issues/new/choose) on GitHub and provide detailed information.
 
 ### I want to contribute
 Check out the [good first issues](https://github.com/openHPI/xikolo-ios/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) or any other unsassigned issue and submit a pull request when you are ready.
 
-# Programming guidelines
+## Programming Guidelines
 
 ### Fastlane
 We use [fastlane](https://github.com/fastlane/fastlane) for repetitive tasks. Have a look at the [fastlane README](fastlane/README.md).
 
-### Code formatting
+#### Useful Commands When Developing
+These custom commands ease the development process.
+
+- `bundle exec fastlane lint`: Report linting warnings for the project (via swiftlint)
+- `bundle exec fastlane format`: Resolve linting warning for the project -- not applicable to all warnings (via swiftlint)
+- `bundle exec fastlane localize`: Extract keys for localized strings and add them to the respective `.strings` files (via BartyCrouch)
+- `bundle exec fastlane check_core_data`: Check if the core data model was modified since the last tagged release
+- `bundle exec fastlane increment_version_(patch|minor|major)`: Increase the app version project-wide in all modules
+- `bundle exec fastlane changelog`: List all commits since the last tagged release
+- `bundle exec fastlane export_localizations`: Export localizations and strip unwanted strings, that excluded by BartyCrouch
+
+### Code Formatting
 In order to have a consistent code formatting, we would like you to set some settings:
-- for less unneccessary whitespace changes please set both checkboxes in Xcode->Preferences->Text Editing regarding whitespaces
-- use Unix-style line endings (LF)
+- For fewer unneccessary whitespace changes, please set both checkboxes in `Xcode > Preferences > Text Editing` regarding whitespaces
```

Comment:
That might become outdated unnoticed with a new Xcode version. Better name these 2 settings explicitly.